### ISSUE TITLE
All days unavailable in agendaDay view

### ISSUE THREAD

**Issue by Caine85** [alextselegidis/easyappointments#436]
Hi! There is a problem in backend when is visualised the day view in the calendar using a non-english language. All days results as unavailable.

The problem is in `/asset/js/backend_calendar_default_view.js`, here:
`var selectedDayName = $calendar.fullCalendar('getView').start.format('dddd').toLowerCase();`

selectedDayName is returned in the selected language and not in english.

So in this line:
`if (workingPlan[selectedDayName] == null) {`

workingPlan[selectedDayName] is always null because there aren't any key in translations that contains a translated string. Instead it's the value for that key that contains the translated string.

For now I resolved this bug in this manner:
```
var selectedDayName = $calendar.fullCalendar('getView').start.format('dddd').toLowerCase();

var weekDays = ['monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday', 'sunday'];
for (var key in weekDays) {
    if(EALang[weekDays[key]].toLowerCase() === selectedDayName) {
        selectedDayName = weekDays[key];
    }
}
```

If it need, I can make a pull request if you think that my solution is adequate.

Thank you so much!

**Comment by alextselegidis** [comment]
@Caine85 You are right on this what, it will be corrected for 1.3.1 

<small>
<table>
<tr>
<td><img src="http://easyappointments.org/img/easyappointments-signature.png"></td>
<td>&nbsp;&nbsp;Alex Tselegidis, Easy!Appointments Creator <br>
&nbsp;&nbsp;<a href="http://alextselegidis.com">Need a customization? Contact me in person!</a></td>
</tr>
</table>
</small>

**Comment by alextselegidis** [comment]
Closed in favor of #461, @Caine85 thank you very much for posting this :) 

<small>
<table>
<tr>
<td><img src="http://easyappointments.org/img/easyappointments-signature.png"></td>
<td>&nbsp;&nbsp;Alex Tselegidis, Easy!Appointments Creator <br>
&nbsp;&nbsp;<a href="http://alextselegidis.com">Need a customization? Contact me in person!</a></td>
</tr>
</table>
</small>
